File: week-07/day-2/Workshop 16.5.2023/animal.py
class Animal:
    hunger = 50
    thirst = 50

    def eat(self):
        Animal.hunger = self.hunger-1

    def drink(self):
        Animal.thirst = self.thirst-1

    def play(self):
        Animal.thirst = self.thirst+1
        Animal.hunger = self.hunger+1


dog = Animal

print(dog.hunger)
print(dog.thirst)
print('----------------------')
dog.eat(dog)

print(dog.hunger)
print(dog.thirst)
print('----------------------')
dog.play(dog)

print(dog.hunger)
print(dog.thirst)
print('----------------------')

File: week-03/day-4/Workshop 18.4/src/Animal/Animal.java
package Animal;

public class Animal {
    private int hunger = 50;
    private int thirst = 50;

    public void eat() {
    hunger--;
        System.out.println("Eating");
    }

    public void drink() {
        thirst--;
        System.out.println("Drinking");
    }
public void play() {
        hunger++;
        thirst++;
    System.out.println("Playing");
}

    public int getHunger() {
        return hunger;
    }

    public void setHunger(int hunger) {
        this.hunger = hunger;
    }

    public int getThirst() {
        return thirst;
    }

    public void setThirst(int thirst) {
        this.thirst = thirst;
    }
}
